src/shared/components/case-editor/services/cases.service.ts

```diff
@@ -240,8 +240,8 @@ export class CasesService {
 
     eventTrigger.wizard_pages.forEach((wizardPage: WizardPage) => {
       wizardPage.parsedShowCondition = new ShowCondition(wizardPage.show_condition);
-      wizardPage.case_fields = this.orderService
-        .deepSort(this.wizardPageFieldToCaseFieldMapper.mapAll(wizardPage.wizard_page_fields, eventTrigger.case_fields));
+      wizardPage.case_fields = this.orderService.sort(
```

Comment:
do we still need to use orderService?

Reply:
Yes the order service still orders the top level fields but the BE is dealing with ordering of compound fields now.